firebase-firestore/ktx/src/main/kotlin/com/google/firebase/firestore/ktx/serialization/FirestoreMapDecoder.kt

```diff
@@ -15,6 +15,10 @@
 package com.google.firebase.firestore.ktx.serialization
 
 import com.google.firebase.firestore.DocumentReference
+import com.google.firebase.firestore.ThrowOnExtraProperties
+import com.google.firebase.firestore.encoding.FirestoreAbstractDecoder as FireDecoder
```

Comment:
I'd rather not use this feature.

Reply:
I have removed this feature by changed the Kotlin's abstract decoder class's name.